json/circe/src/main/scala/sttp/tapir/json/circe/TapirJsonCirce.scala

```diff
@@ -27,11 +27,11 @@ trait TapirJsonCirce {
 
   def jsonPrinter: Printer = Printer.noSpaces
 
-  // Json is a coproduct with unknown implementations
+  // Json is a product with unknown implementations
   implicit val schemaForCirceJson: Schema[Json] =
     Schema(
-      SCoproduct(Nil, None)(_ => None),
-      Some(SName("io.circe.Json"))
+      SProduct(Nil),
```

Comment:
what about other json integrations? Play-json, json4s etc., I think they also contain schemas for arbitrary json values

Reply:
on second though ... I'm not sure if `Product` is the right type here. A `Json` value can be an object OR a number OR a string etc. - hence the original mapping of `SCoproduct`, meaning that it can be anythng really. So - can we make it work with `SCoproduct`?